semih-week2.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
Instead of putting the homework in a separate file you should edit the files in the homework folder. Look at the files at `Week2/homework/maartjes-work.js` and `Week2/homework/map-filter.js`. Look for this comment `// Replace this comment and the next line with your code`.  If you put your code in these files and run `npm run test-week2` in your terminal you can check if the test passes or not.